klippy/kinematics/polar.py

```diff
@@ -108,7 +108,17 @@ def check_move(self, move):
     def get_status(self, eventtime):
         xy_home = "xy" if self.limit_xy2 >= 0. else ""
         z_home = "z" if self.limit_z[0] <= self.limit_z[1] else ""
-        return {'homed_axes': xy_home + z_home}
+        return {
+            'homed_axes': xy_home + z_home,
+            'axis_minimum': {
+                'x': -math.sqrt(self.limit_xy2),
+                'y': -math.sqrt(self.limit_xy2),
+                'z': self.limit_z[0]},
+            'axis_maximum': {
+                'x': math.sqrt(self.limit_xy2),
+                'y': math.sqrt(self.limit_xy2),
+                'z': self.limit_z[1]}
```

Comment:
The limits on polar should also come from `self.rails[0].get_range()` and `self.rails[1].get_range()`.